docs/source/prometheus.rst

```diff
@@ -40,6 +40,12 @@ dask_scheduler_tasks_forgotten_total
        Task groups on the scheduler which have all tasks in the forgotten state are not
        included.
 
+dask_scheduler_tasks_time_spent_compute_seconds_total
+    Total time spent computing a task
+dask_scheduler_tasks_time_spent_transfer_seconds_total
+    Total time spent transferring a task
+dask_scheduler_tasks_nbytes_bytes_total
+    The total number of bytes that this task prefix has produced
```

Comment:
```suggestion
dask_scheduler_tasks_time_spent_compute_seconds_total
    Total time (per prefix) spent computing tasks
dask_scheduler_tasks_time_spent_transfer_seconds_total
    Total time (per prefix) spent transferring
dask_scheduler_tasks_nbytes_bytes_total
    The total number of bytes that this task prefix has produced
```

Might be good to be explicit that this is broken down by prefix (and not, say, individual task or task group).